src/main/kotlin/no/nav/familie/ef/sak/api/gui/PersonopplysningerController.kt

```diff
@@ -0,0 +1,31 @@
+package no.nav.familie.ef.sak.api.gui
```

Comment:
Usikker på bruk av `gui` i pakkene. 
Strengt tatt kommer jo det aller meste av api-er i sak til å lages for GUI.